src/Components/SalesforceWrapper.tsx

```diff
@@ -18,13 +18,8 @@ export const SalesforceWrapper: React.FC<SalesforceWrapperProps> = ({
   let chatURL = null
 
   if (salesforceMessageEnabled) {
-    if (isInAuction) {
-      serviceName = getENV("SALESFORCE_AUCTION_MESSAGE_SERVICE_NAME")
-      chatURL = getENV("SALESFORCE_AUCTION_MESSAGE_INSTANCE_URL")
-    } else {
-      serviceName = getENV("SALESFORCE_COLLECTOR_MESSAGE_SERVICE_NAME")
-      chatURL = getENV("SALESFORCE_COLLECTOR_MESSAGE_INSTANCE_URL")
-    }
+    serviceName = getENV("SALESFORCE_MESSAGE_SERVICE_NAME")
+    chatURL = getENV("SALESFORCE_MESSAGE_INSTANCE_URL")
```

Comment:
We can't delete it yet, because it's not toggled on and is currently used in other code, but definitely want to remove it and I will have to update my [clean up PR](https://github.com/artsy/force/pull/14811) to do this

Reply:
Ahhh cool, all good then 👌 ✨